src/libs/antares/study/parts/short-term-storage/container.cpp

```diff
@@ -102,6 +102,10 @@ bool STStorageInput::LoadConstraintsFromIniFile(const fs::path& parent_path)
                 value.to<std::string>(clusterName);
                 additional_constraints.cluster_id = transformNameIntoID(clusterName);
             }
+            else if (key == "enabled")
+            {
+                value.to<bool>(additional_constraints.enabled);
+            }
```

Comment:
if it's disabled, why bother to read and store it?

Reply:
Possible reasons
- GUI/save => no longer the case
- Indexes => we want to keep the same indices when disabling an object for reproducibility reasons

But we would need to look for the "enabled" key first, and decide later. Another dev would be required

Reply:
I suggest that we skip rhs & finding the STS if the constraint is disabled